--- tar-1.23/configure.ac~	2010-04-26 09:26:23.000000000 +0300
+++ tar-1.23/configure.ac	2010-04-26 09:26:25.460605965 +0300
@@ -24,10 +24,7 @@
 AC_CONFIG_AUX_DIR([build-aux])
 AC_CONFIG_HEADERS([config.h])
 AC_PREREQ([2.63])
-AM_INIT_AUTOMAKE([1.11 gnits tar-ustar dist-bzip2 dist-shar std-options silent-rules])
-
-# Enable silent rules by default:
-AM_SILENT_RULES([yes])
+AM_INIT_AUTOMAKE([1.10 gnits tar-ustar dist-bzip2 dist-shar std-options])
 
 AC_PROG_CC
 AC_EXEEXT
