--- tar-1.34/src/list.c.orig	2021-03-06 10:21:12.348727417 +0100
+++ tar-1.34/src/list.c	2021-03-06 15:12:39.174304779 +0100
@@ -241,6 +241,14 @@
 
 	  if (!ignore_zeros_option)
 	    {
+		break;
+
+      /*
+       * According to POSIX tar specs, this is wrong, but on the web
+       * there are some tar specs that can trigger this, and some tar
+       * implementations create tars according to that spec...
+       *
+
 	      char buf[UINTMAX_STRSIZE_BOUND];
 
 	      status = read_header (&current_header, &current_stat_info,
@@ -251,6 +259,7 @@
 		       (0, 0, _("A lone zero block at %s"),
 			STRINGIFY_BIGINT (current_block_ordinal (), buf)));
 	      break;
+       */
 	    }
 	  status = prev_status;
 	  continue;
