diff -ur tar-1.13.orig/src/buffer.c tar-1.13/src/buffer.c
--- tar-1.13.orig/src/buffer.c	Mon Jul  5 08:47:59 1999
+++ tar-1.13/src/buffer.c	Wed Aug 25 22:18:55 1999
@@ -893,6 +893,7 @@
 {
   int copy_back;
   ssize_t status;
+  int written;
 
   if (checkpoint_option && !(++checkpoint % 10))
     WARN ((0, 0, _("Write checkpoint %d"), checkpoint));
@@ -905,6 +906,15 @@
     }
   else if (dev_null_output)
     status = record_size;
+  else if (S_ISFIFO(archive_stat.st_mode)) {/*ppe: fix writing over a pipe */
+    status=0;
+    written=0;
+    while (status != record_size && written >=0 ) {
+      written=rmtwrite (archive, record_start->buffer+status,
+		       (unsigned int) (record_size-status));
+      if (written>=0) status += written;
+    }
+  }
   else
     status = write_archive_buffer ();
   if (status != record_size && !multi_volume_option)
