--- tar-1.13.93/configure.ac.orig	2004-03-02 00:19:00.000000000 +0100
+++ tar-1.13.93/configure.ac	2004-03-02 00:34:44.077767376 +0100
@@ -277,8 +277,9 @@
                                           [Conversion descriptor type]),
                                 [#include <iconv.h>])])
 
+AC_CHECK_FUNC(iconv, [AC_DEFINE([HAVE_LIBICONV])],[
 AC_CHECK_LIB(iconv, iconv)
-
+])
 
 AC_OUTPUT([Makefile m4/Makefile \
            doc/Makefile\
