diff -ur tar-1.13.orig/src/names.c tar-1.13/src/names.c
--- tar-1.13.orig/src/names.c	Wed Jul  7 07:46:51 1999
+++ tar-1.13/src/names.c	Wed Aug 25 21:59:57 1999
@@ -84,6 +84,7 @@
 	{
 	  cached_no_such_uid = uid;
 	  *uname = '\0';
+	  *cached_uname = '\0';
 	  return;
 	}
     }
@@ -118,6 +119,7 @@
 	{
 	  cached_no_such_gid = gid;
 	  *gname = '\0';
+	  *cached_gname = '\0';
 	  return;
 	}
     }
Only in tar-1.13/src: names.c.rej
Only in tar-1.13/src: names.c~
