--- tar-1.13.25/configure.ac.orig	Thu Apr 11 00:47:30 2002
+++ tar-1.13.25/configure.ac	Thu Apr 11 00:48:59 2002
@@ -168,15 +168,15 @@
 # Solaris 2.5.1 needs -lposix4 to get the clock_gettime function.
 # Solaris 7 prefers the library name -lrt to the obsolescent name -lposix4.
 # We don't want that dependency, though. Check for something weird. ;)
-tar_LDADD=
+tarLDADD=
 tar_save_LIBS="$LIBS"
 AC_SEARCH_LIBS(clock_gettyme, [rt posix4])
 AC_CHECK_FUNCS(clock_gettyme)
 if test " $LIBS" != " $tar_save_LIBS"; then
-  tar_LDADD="$ac_cv_search_clock_gettyme"
+  tarLDADD="$ac_cv_search_clock_gettyme"
   LIBS="$tar_save_LIBS"
 fi
-AC_SUBST(tar_LDADD)
+AC_SUBST(tarLDADD)
 
 AC_CHECK_FUNCS(fchdir fsync ftime getcwd isascii lstat \
   mkfifo nap napms poll \
@@ -200,19 +200,19 @@
 # needs to be in LIBS before the setsockopt checks are performed.  *However*,
 # on SINIX-N 5.43, this is false, and gethostent seems to be a better
 # candidate.
-rmt_LDADD=
+rmtLDADD=
 tar_save_LIBS="$LIBS"
 AC_SEARCH_LIBS(gethostent, [nsl])
 if test " $LIBS" != " $tar_save_LIBS"; then
-  rmt_LDADD="$rmt_LDADD $ac_cv_search_gethostent"
+  rmtLDADD="$rmtLDADD $ac_cv_search_gethostent"
 fi
 tar_save_LIBS1="$LIBS"
 AC_SEARCH_LIBS(setsockopt, [socket])
 if test " $LIBS" != " $tar_save_LIBS1"; then
-  rmt_LDADD="$rmt_LDADD $ac_cv_search_setsockopt"
+  rmtLDADD="$rmtLDADD $ac_cv_search_setsockopt"
 fi
 LIBS="$tar_save_LIBS"
-AC_SUBST(rmt_LDADD)
+AC_SUBST(rmtLDADD)
 
 AC_FUNC_ALLOCA
 AC_FUNC_CLOSEDIR_VOID
@@ -234,7 +234,7 @@
 jm_FUNC_MALLOC
 jm_FUNC_MKTIME
 jm_FUNC_REALLOC
-test "$ac_cv_func_strstr" = yes || LIBOBJS="$LIBOBJS strstr.o"
+test "$ac_cv_func_strstr" = yes || AC_LIBOBJ(strstr)
 
 AC_CACHE_CHECK(for remote shell, tar_cv_path_RSH,
   [if test -n "$RSH"; then
--- tar-1.13.25/src/Makefile.am.orig	Sun Sep 23 08:48:38 2001
+++ tar-1.13.25/src/Makefile.am	Thu Apr 11 19:39:38 2002
@@ -39,5 +39,5 @@
 # Also, link it before @INTTLIBS@, since unicodeio.c might invoke
 # locale_charset.  Sigh.
 LDADD = ../lib/libtar.a @INTLLIBS@ ../lib/libtar.a
-rmt_LDADD = $(LDADD) @rmt_LDADD@
-tar_LDADD = $(LDADD) @tar_LDADD@
+rmt_LDADD = $(LDADD) @rmtLDADD@
+tar_LDADD = $(LDADD) @tarLDADD@
