--- tar-1.13.92/configure.ac.orig	2003-12-12 15:32:46.000000000 +0100
+++ tar-1.13.92/configure.ac	2003-12-13 11:07:47.008259632 +0100
@@ -152,10 +152,6 @@
 # Solaris 7 prefers the library name -lrt to the obsolescent name -lposix4.
 tar_save_LIBS=$LIBS
   LIB_CLOCK_GETTIME=
-  AC_SEARCH_LIBS(clock_gettime, [rt posix4])
-  case "$ac_cv_search_clock_gettime" in
-    -l*) LIB_CLOCK_GETTIME=$ac_cv_search_clock_gettime;;
-  esac
   AC_SUBST(LIB_CLOCK_GETTIME)
   AC_CHECK_FUNCS(clock_gettime)
 LIBS=$tar_save_LIBS
